app/src/main/java/com/andrew/tasky/agenda/presentation/screens/event_detail/EventDetailFragment.kt

```diff
@@ -525,4 +578,19 @@ class EventDetailFragment : Fragment(R.layout.fragment_event_detail) {
             EventDetailFragmentDirections.actionEventDetailFragmentToPhotoDetailFragment()
         )
     }
+
+    @RequiresApi(Build.VERSION_CODES.TIRAMISU)
+    private fun areNotificationPermissionsEnabled(context: Context): Boolean {
```

Comment:
Wouldn't `isNotificationPermissionGranted` make more sense for the name, since it's not multiple permissions? (and because permissions aren't really enabled or disabled, but rather granted or not granted)